### PR TITLE
feat: added feature1

### PR DESCRIPTION
**Pull request by haileymandal**
Updated a to 16 and b to 12 for a total expected 28

### CODE REVIEW

File: src/sum.test.js [hunk 1]
```diff
@@ -2,13 +2,13 @@
 const sum = require('./sum')
 
 describe('sum()', () => {
-  test('Given 5 and 2, should sum 7', () => {
+  test('Given 16 and 12, should sum 28', () => {
     const given = {
-      a: 5,
-      b: 2
+      a: 16,
```
Comment (neerajmandal) [inline src/sum.test.js:7]: Great idea!

File: src/sum.test.js [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
+      b: 12
     }
 
-    const expected = 7
+    const expected = 28
```
Comment (neerajmandal) [inline src/sum.test.js:11]: Keep it going girl

File: src/sum.test.js (hunk 1 above)
Comment (haileymandal) [reply src/sum.test.js:7]: thanks bb

File: src/sum.test.js (hunk 2 above)
Comment (haileymandal) [reply src/sum.test.js:11]: I'll try ;D

**Review by neerajmandal — APPROVED** [review]

**Pull request merged**